analyze-ms-stats/calc-manuscript-stats.sh

```diff
@@ -0,0 +1,19 @@
+# Calculate growth statistics for manuscript based on Manubot files
```

Comment:
A minor suggestion, in case you want the script to stop if any command fails (instead of continuing).

```suggestion
#!/bin/bash
set -e

# Calculate growth statistics for manuscript based on Manubot files
```